index.html

```diff
@@ -9,39 +9,39 @@
     </head>
 
     <body>
-
-        <div class="instagram-wrapper"> 
-            <div class="instagram-phone">
-                <img src="./img/instagram-celular.png" alt="celular">
-            </div>       
-            <div class="instagram-continue">
-                <div class="group">
-                <img src="./img/instagram-logo.png" class="instagram-logo" alt="instagram-logo">
-                <div class="profile-photo">
-                    <img src="./img/perfil-instagram.jpg" alt="foto de perfil">
-                </div>
-
-                <a href="#" class="instagram-login">Continue como eubeanunes</a>
-                <a href="#" class="instagram-logout">Remover conta</a>
-                </div>
+        <main class="instagram-wrapper"> 
+            <article class="instagram-continue">
+                <img class="instagram-phone" src="./img/instagram-celular.png" alt="iphone demonstrando o aplicativo Instagram">
+                <form class="instagram-form" action="">
```

Comment:
Em teoria você tem um formulário na hora de fazer login. Como a gente precisa de um elemento para controlar o layout dos grupos, o melhor é usar o elemento `form` ao invés de mais um `div` gratuíto.

Reply:
O mais interessante é que eu também fiz esse projeto dessa mesma aula e praticamente replicamos a professora, sendo que antes eu já tinha visto esse conceito de criar formulário, mas não pensei em aplicar assim. Isso indica que preciso estudar mais e aplicar as tags necesárias. Muito obrigada!